contracts/test-utils/package.json

```diff
@@ -0,0 +1,76 @@
+{
+    "name": "@0x/contracts-test-utils",
+    "version": "1.0.0",
+    "engines": {
+        "node": ">=6.12"
+    },
+    "description": "Test utils for 0x contracts",
+    "main": "lib/src/index.js",
+    "directories": {
+        "test": "test"
+    },
+    "scripts": {
+        "build": "tsc -b",
+        "build:ci": "yarn build",
+        "test": "yarn run_mocha",
+        "test:coverage": "run-s build run_mocha coverage:report:text coverage:report:lcov",
+        "run_mocha": "mocha --require source-map-support/register --require make-promises-safe 'lib/test/**/*.js' --timeout 100000 --bail --exit",
+        "clean": "shx rm -rf lib",
+        "lint": "tslint --format stylish --project tsconfig.lint.json",
+        "coverage:report:text": "istanbul report text",
+        "coverage:report:html": "istanbul report html && open coverage/index.html",
+        "profiler:report:html": "istanbul report html && open coverage/index.html",
+        "coverage:report:lcov": "istanbul report lcov",
```

Comment:
Are there tests in this package? Do we need converage?

Reply:
Yes. There is one test file. Testing test with refrence test util. Why not?